docs/modules/ROOT/pages/index.adoc

```diff
@@ -2,7 +2,9 @@
 :imagesdir: ../assets/images
 :!sectids:
 
-OpenNMS Helm Charts is a project that makes it easier for users to run OpenNMS Horizon in a https://kubernetes.io/[Kubernetes] cluster.
+OpenNMS Helm Charts makes it easier for users to run OpenNMS Horizon locally in a https://kubernetes.io/[Kubernetes] cluster or Red Hat OpenShift.
+It is not yet available for a Cloud environment.
```

Comment:
Technically, they can run our Helm Charts on an environment running in a cloud environment (for example OpenShift environment running on Azure)